### DISCUSSION TITLE
Adding support to visidata.

### DISCUSSION THREAD

**Discussion by Awannaphasch2016** [emacs-eaf/emacs-application-framework#870] [Ideas]
I am a big fans of visidata. (data tangling tools)
Both use python.

I discorvered eaf recently, and want to work on a project that would give me deeper understanding of visidata, emacs, elisp, so I figure this should be a fun projects.

anyone is welcomed to join on the project.
any advice, comment, thought?

discord community: https://discord.gg/a6HGZz5eya

**Comment by MatthewZMD** [comment]
Have fun hacking 😄
